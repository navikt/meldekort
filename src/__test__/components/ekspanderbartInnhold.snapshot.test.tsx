import * as React from "react";
import * as renderer from "react-test-renderer";
import ProviderWrapper from "../testSetup/providerWrapper";
import EkspanderbartInnhold from "../../app/components/ekspanderbartInnhold/ekspanderbartInnhold";

it("EkspanderbartInnhold", () => {
  const tree = renderer.create(
    <ProviderWrapper>
      <EkspanderbartInnhold children={<span>Test</span>} />
    </ProviderWrapper>
  );
  expect(tree).toMatchSnapshot();
});
