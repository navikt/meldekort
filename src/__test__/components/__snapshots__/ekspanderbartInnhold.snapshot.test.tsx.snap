// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EkspanderbartInnhold 1`] = `
<div
  className="ekspanderbartInnhold"
  style={
    Object {
      "height": "0px",
      "overflow": "hidden",
    }
  }
>
  <div
    className="ReactCollapse--content"
  >
    <div
      aria-live="off"
    >
      <div />
    </div>
  </div>
</div>
`;
